### ISSUE TITLE
Bug: Ender 3v2 auto-home 'broken' after power-off during filament feed/filament load command

### ISSUE THREAD

**Issue by Hyenadae** [mriscoc/Marlin_Ender3v2#127]
### Description

I've been using this firmware for a few weeks now without issue and generally worked better than the creality stock firmware. Especially now that I have a BLTouch. But for some reason, my auto-home (specifically the Y-axis) command and routine does not work as expected after I turned the printer off as it was loading filament by accident (as I didn't want to waste more). 

Now, when auto-home occurs, it behaves like normal until actually moving to the position that is marked 'home'. In the past, X=156 Y=130/120 (130 is creality stock) position indicated would be right in the middle of the bed, where you would expect and is where the printer would put the extruder/probe with the stock firmware. Now, it goes to essentially the end of the printable bed region, just above or at the 'N' in Ender (Ender 3v2 stock bed) every time.

If I use the new firmware's 'Home Y', it hits the Y-Stop as expected and then I can use the manual-move menu to place it at Y=122 (nozzle over bed center, which is good), which is where it was in the past until this issue occurred. Even if I save settings, and reboot, doing the 'auto home' option (and at the start of prints) will bring the wrong placement and problem back.

Possible other issue is that after this glitch, some unknown setting is stored somewhere on either EEPROM or SDcard (re-formatted and kept restoring defaults + rebooting printer but this did not fix it)

**Expected behavior:** [What you expect to happen]
Use Auto-Home, probe and extruder go to the center of the bed and the bed position matches what you expect from the position readout after homing. 

**Actual behavior:** [What actually happens]

See above. It doesn't anymore, persists through various EEPROM wipe + save + default commands, even after flashing the stock Creality BLTouch firmware (which restores the printer's auto-home to original behavior). 

#### Additional Information
TLDR: 

Started out with Creality FW 1.0.2 (No BLTouch)
Upgraded to your firmware, early Sept-2021 build with/and installed BLTouch (V4.22 board)
No issues with autohome positioning or Y-axis bed misreadings until unexpected power off during a menu-commanded filament-feed. On reboot, auto-home on the Y-Axis was not centered, and refused to center after various troubleshooting steps.
Printer hardware has not changed, stock FW works. Z and X axis auto-homing works fine though.

No idea what else to do aside from use stock Creality firmware or find a way to truly force whatever settings or positions the stock FW uses or has into this new firmware's EEPROM settings

**Comment by mriscoc** [comment]
Thank you for the feedback, be careful about layer shift on Y axis.
